### ISSUE TITLE
Download resources results not announced to screen reader after filtering (WCAG Level AA)

### ISSUE THREAD

**Issue by jrbarnes9** [ONSdigital/design-system#2243]
Add `role="status"` to the number of results div which changes after filtering, to announce to screen readers.

Page 58-59 of June 2022 DAC DS report...

----
## Status Message
Screen reader users are not informed when more content has been added to the page.
WCAG Reference:
4.1.3 Status Messages (Level AA)
Understanding Status Messages |How to Meet Status Messages

### Issue ID: DAC_Status_Message_01

URL: https://ons-design-system.netlify.app/patterns/download-resources/examples/download-resources/
Page title: Service name
Journey: Journey 2, Step 2

Changing the state of checkboxes within the filters section of the page results in the text ‘9 results of 150’ updating to reflect the updating results. The change of results text is not audibly read to screen reader users, meaning that the user may be unaware that results have updated.

### Current code ref(s):
```
#main-content > div.ons-js-adv-filter > div > div.ons-grid__col.ons-col-8\@m > div.ons-adv-filter__results-options > div.ons-adv-filter__results-count <div class="ons-adv-filter__results-count"> <span class="ons-js-adv-filter__results-count">9</span> results of 150 </div>
```

### Solution:
Results information that changes once the user has selected a filters option must be audibly announced for screen reader users to be aware that content has updated. To audibly announce the results text, an attribute role=status must be used. The role=status attribute must be added to the main container of the text that is to be announced when the page loads; this means that any change to that div will announce.
